R/ForecastRegrTask.R

```diff
@@ -0,0 +1,58 @@
+#' @title Create a task for univariate forecasting
+#'
+#' @rdname Task
+#'
+#' @description Creates a task for univariate forecasting learners
+#'
+#' @export
+makeForecastRegrTask = function(id = deparse(substitute(data)), data, target,
+  weights = NULL, blocking = NULL, frequency = 1L, fixup.data = "warn",
+  check.data = TRUE) {
+  assertString(id)
+  assertClass(data, "xts")
```

Comment:
> Maybe it would be better to forget about order() and just throw an error if dates are not in order.

This could be included in `fixup.data` and we will throw a warning if they are out of order